session-3-jshell/README.md

```diff
@@ -4,8 +4,33 @@ Java REPL (Read-Eval-Print-Loop): previously called Kulla.
 
 The JShell API can be used via applications as well.
 
-Look for the the JShell REPL in the `JAVA_HOME/bin` folder:
+Windows Users
 
+    Look for the the JShell REPL in the `JAVA_HOME\bin` folder:
+
+    c:\... dir /W %JAVA_HOME%\bin\js*
+    
+    Directory de C:\Program Files\Java\jdk-9\bin
```

Comment:
Oh, Yes You are right, I used a windows in Spanish and forgot to translate "de" for of